lua/hbac/telescope/init.lua

```diff
@@ -0,0 +1,93 @@
+local M = {}
```

Comment:
```suggestion
if not pcall(require, "telescope") then
	local msg = "Please install telescope.nvim to use hbac's telescope picker"
	vim.notify(msg, vim.log.levels.ERROR, { title = "Hbac" })
	return
end

local M = {}
```

Since plenary is a dependency for telescope and it's only used as part of the telescope module, I didn't add it. Should it be there anyway?